otx/core/data/adapter/action_dataset_adapter.py

```diff
@@ -83,7 +83,7 @@ def _prepare_label_information(self, datumaro_dataset: dict) -> Dict[str, Any]:
 
         """
         outputs = {
-            "label_entities": [],
+            "label_entities": {},
```

Comment:
is this change mandatory for the apache arrow support? It looks like more about improvement.

Reply:
It was mandatory before, but the architecture of arrow dataset has been changed. I will test if reverting it would make no difference.

Reply:
`label entity` is reverted back to list in https://github.com/openvinotoolkit/training_extensions/pull/2009/commits/7b040d151fd5f0b8944ece5ebbed225eefabfda5